recipes/esmf/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ $(uname) == Linux ]]; then
+yum install -y gcc-gfortran
```

Comment:
gfortran is in the new Docker image.  One thing we need to keep in mind when building package which have Fortran code is that they will have a dependency on libgfortran unless the package is configuration statically link the the gfortran runtime.  Similar to how the setup of gcc defines which version of libc (and libgcc_s) the version of the gfortran is also set by the compiler and needs to be watched.